core/sched.c

```diff
@@ -118,7 +118,14 @@ int __attribute__((used)) sched_run(void)
 #ifdef MODULE_SCHEDSTATISTICS
         schedstat *active_stat = &sched_pidlist[active_thread->pid];
         if (active_stat->laststart) {
-            active_stat->runtime_ticks += now - active_stat->laststart;
+            /* ensure now > laststart, to avoid flawed sum of runtime ticks
```

Comment:
> ... losing 71minute overflows ...

It would mean that all running/sleeping periods would be counted module ```2**32``` xtimer ticks, which is ~71minutes. I doubt that it would have significant impact in ps output, compared to dropping all periods that happen to have an overflow.